frontend/src/components/GeneratePass/generatePass.js

```diff
@@ -48,19 +49,25 @@ const GeneratePass = () => {
   const generatePassHandler = async (e) => {
     e.preventDefault()
     setFormErrors(validateForm(newPass))
-    // console.log(localStorage.getItem("token"))
-    const res = await axios.post(
-      "http://localhost:5000/passes/generatePass/",
-      newPass,
-      {
-        headers: { authorization: "Bearer " + localStorage.getItem("token") },
+    try {
+      setError("")
+      const res = await axios.post(
+        "https://backend-dun-nine.vercel.app/passes/generatePass/",
```

Comment:
Didn't you need to change it when you start development? And similarly I guess you need to change when you are going to deploy. Look a work around to it.

Reply:
by yarn start it is working locally so i don't need to change it everytime

Reply:
And do you need to change that when you deploy it?

Reply:
no

Reply:
Lets' discuss it.

Reply:
on call